public/javascripts/components/header.js

```diff
@@ -0,0 +1,54 @@
+/////////////////////////////// Header ///////////////////////////////
+  var Header = React.createClass({
+    render: function() {
+      return (
+        <div className="header">
+        <div className="container-fluid">
+          {/* Brand and toggle get grouped for better mobile display */}
+          <div className="navbar-header">
+            <button data-target="#bs-example-navbar-collapse-1" data-toggle="collapse" className="navbar-toggle collapsed" type="button">
+              <span className="sr-only">Toggle navigation</span>
+              <span className="icon-bar" />
+              <span className="icon-bar" />
+              <span className="icon-bar" />
+            </button>
+          </div>
+          {/* Collect the nav links, forms, and other content for toggling */}
+          <div id="bs-example-navbar-collapse-1" className="collapse navbar-collapse">
+            <ul className="nav navbar-nav">
+              <li><a href="/">Home <span className="sr-only">(current)</span></a>
+              </li>
+              <li><a href="/quotes">Quotes</a>
+              </li>
+              <li className="active"><a href="/authors">Authors</a>
```

Comment:
This is wrong. You are setting the active class for all views to /authors. You should switch the active class depending on the current view.

You can pass a parameter view from the backend to the client to highlight the current view.